Userland/Libraries/LibVT/TerminalWidget.cpp

```diff
@@ -1135,6 +1140,54 @@ void TerminalWidget::update_paste_action()
     m_paste_action->set_enabled(GUI::Clipboard::the().mime_type().starts_with("text/") && !GUI::Clipboard::the().data().is_empty());
 }
 
+void TerminalWidget::set_color_scheme(const StringView& name)
+{
+    if (name.contains('\\')) {
+        dbgln("Shenanigans! Color scheme name contains a slash, which is not a valid file name.");
+        return;
+    }
+
+    m_color_scheme_name = name;
+
+    constexpr StringView color_names[] = {
+        "Black",
+        "Red",
+        "Green",
+        "Yellow",
+        "Blue",
+        "Magenta",
+        "Cyan",
+        "White"
+    };
+
+    auto color_config = Core::ConfigFile::open(String::formatted("/res/terminal_colors/{}.ini", name));
```

Comment:
Let's not use an underscore in the path here, that seems pretty uncommon across the system. `terminal-colors` looks less out of place.